sources/cassandra/main.tf

```diff
@@ -0,0 +1,102 @@
+resource "google_compute_firewall" "allow_ports" {
+  name    = "allow-zdm-ports"
+  network = var.common_params.network
+
+  allow {
+    protocol = "tcp"
+    ports    = ["14002", "9042"]
+  }
+
+  source_ranges = ["0.0.0.0/0"]
+  target_tags   = ["zdm"]
+}
+
+resource "google_compute_instance" "cos_instances" {
+  count        = var.common_params.instance_count
+  name         = "${var.common_params.instance_name_prefix}-${count.index}"
+  machine_type = var.common_params.machine_type
+  zone         = var.common_params.zone
+
+  boot_disk {
+    initialize_params {
+      image = "projects/cos-cloud/global/images/cos-105-17412-535-55"
+      size  = 32
+      type  = "pd-balanced"
```

Comment:
If we are not using local storage for anything performance critical, would `pd-standard` give the customer better price point?